src/sound.h

```diff
@@ -46,6 +48,12 @@ struct SimpleSoundSpec
 	float gain = 1.0f;
 	float fade = 0.0f;
 	float pitch = 1.0f;
+	float offset_start = 0.0f;
+	float offset_end   = 1.0f;
```

Comment:
This is a remnant of older code which represented offset units as fractions from [0...1] instead of units as seconds of [0...duration of sound file]. Luckily it is not a bug, as -1 is correctly used in the constructor. Will fix.